composer.json

```diff
@@ -79,6 +79,7 @@
       "SilverStripe\\Security\\Tests\\": "tests/php/Security/",
       "SilverStripe\\View\\": "src/View/",
       "SilverStripe\\View\\Tests\\": "tests/php/View/",
+      "SilverStripe\\View\\ViewSupport\\Tests\\": "tests/php/View/ViewSupport",
```

Comment:
Namespace should be `SilverStripe\View\Tests\<others>` and thus not need a separate composer.json declaration.